lib/diagnostic.js

```diff
@@ -6,26 +6,50 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const splitWords = function (paragraphString){
+  let words = paragraph.split(' ');
```

Comment:
It's yelling at me that "paragraph" isn't defined. Why isn't paragraph defined here?